play/apps/game/engine.py

```diff
@@ -13,19 +14,22 @@ def run(config):
 
 
 def status(id):
+    # Import here to prevent circular import issue
+    from apps.game.models import Game
+
     res = requests.get(f'{settings.ENGINE_URL}/games/{id}')
     res.raise_for_status()
     data = res.json()
 
-    status = data['Game'].get('Status', 'pending')
+    status = data['Game'].get('Status', Game.Status.PENDING)
     turn = data['LastFrame'].get('Turn', 0)
     gsnakes = data['LastFrame'].get('Snakes', [])
 
     snakes = {}
     for s in gsnakes:
         id = s['ID']
         snakes[id] = {
-            'death': s['Death']['Cause'] if s['Death'] else '',
+            'death': s['Death']['Cause'] if s['Death'] else Game.Status.PENDING,
```

Comment:
This fixes a bug @joram and I discovered where you can't create more than 1 bracket game because @brandonb927 was a silly goose a few months ago.